CMakeLists.txt

```
@@ -227,13 +211,7 @@ if (NOT MINGW)
 endif()
 
 target_compile_definitions(${PROJECT} PRIVATE _NO_LOCO_WIN32_=1)
-if (USE_BOOST_FILESYSTEM)
-    target_link_libraries(${PROJECT} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY})
-    target_include_directories(${PROJECT} PRIVATE ${Boost_INCLUDE_DIRS})
-    target_compile_definitions(${PROJECT} PRIVATE BOOST_SYSTEM_NO_DEPRECATED=1 BOOST_FILESYSTEM_NO_DEPRECATED=1 _OPENLOCO_USE_BOOST_FS_=1)
-else()
-	target_link_libraries(${PROJECT} stdc++fs)
-endif()
+target_link_libraries(${PROJECT} stdc++fs)
```

This needs to be disabled on macOS, e.g.:

```suggestion
if (NOT APPLE)
    target_link_libraries(${PROJECT} stdc++fs)
endif()
```